sapl/templates/sessao/blocos_ata/expedientes.html

```diff
@@ -0,0 +1,8 @@
+<fieldset>
+    <p align="justify">
+        <strong>Expedientes: </strong> {% for e in expedientes %}
```

Comment:
A formatação não tá legal. O interessante é alinhar o `for` com o final dele. Assim:
```
<strong>Expedientes: </strong>
        {% for e in expedientes %}
              <b>{{e.tipo}}</b>:
                 {{e.conteudo|striptags}}
        {% endfor %}
```